packages/odyssey-react/package.json

```diff
@@ -36,7 +32,13 @@
     "babel-jest": "^26.6.3",
     "jest": "^26.6.3",
     "jest-axe": "^5.0.1",
-    "postcss": "^8.3.6"
+    "postcss": "^8.3.6",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "peerDependencies": {
+    "react": ">=16.14",
+    "react-dom": ">=16.14"
```

Comment:
Seeing the changes here, are we not using an upper bound because we support React 17 too?